api/internal/handlers/kubernetes/install.go

```diff
@@ -171,3 +171,24 @@ func (h *Handler) PostSetAppConfigValues(w http.ResponseWriter, r *http.Request)
 
 	h.GetAppConfig(w, r)
 }
+
+// GetAppConfigValues handler to get the app config values
+//
+//	@ID				getKubernetesInstallAppConfigValues
+//	@Summary		Get the app config values
+//	@Description	Get the app config values
+//	@Tags			kubernetes-install
+//	@Security		bearerauth
+//	@Produce		json
+//	@Success		200	{object}	map[string]string
+//	@Router			/kubernetes/install/app/config/values [get]
+func (h *Handler) GetAppConfigValues(w http.ResponseWriter, r *http.Request) {
+	configValues, err := h.installController.GetAppConfigValues(r.Context())
+	if err != nil {
+		utils.LogError(r, err, h.logger, "failed to get app config values")
+		utils.JSONError(w, r, err, h.logger)
+		return
+	}
+
+	utils.JSON(w, r, http.StatusOK, configValues, h.logger)
```

Comment:
I think you should make this a type similar to 

```
type SetAppConfigValuesRequest struct {
	Values map[string]string `json:"values"`
}
```

Reply:
hmm, that one is a request, this is a response. making it a separate type would cause the `values` to be nested.

Reply:
I think we should nest this in a type to make the api cleaner